eng/common/scripts/job-matrix/job-matrix-functions.ps1

```diff
@@ -380,7 +380,7 @@ function ProcessImport([MatrixParameter[]]$matrix, [String]$selection, [Array]$n
     }
 
     if (!(Test-Path $importPath)) {
-        Write-Error "`$IMPORT path '$importPath' does not exist."
+        Write-Error "`$IMPORT path '$importPath' does not exist. Current dir: $(Get-Location)"
```

Comment:
Is this debugging information something you plan to sync in eng/common or is it only temporary to help with getting these tests working?